pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1820,6 +1831,23 @@
         ^=        rcvr  her
         ^=   rcvr-life  her-life
     ==
+  ::  +sendkeys-packet: generate a request for a self-attestation.
+  ::
+  ::    Sent by non-comets to comets.  Not acked.
+  ::
+  ++  sendkeys-packet
+    |=  [her=ship]
+    ^-  blob
+    ?>  ?=(%pawn (clan:title her))
+    %-  encode-packet
+    %-  encode-shut-packet
+    :*  ^=    shut-packet  *shut-packet
+        ^=  symmetric-key  *symmetric-key
+        ^=           sndr  our
+        ^=           rcvr  her
+        ^=      sndr-life  life.ames-state
+        ^=      rcvr-life  1
```

Comment:
I wonder if these lives should both be `0`, since normal packets will never have that.

Reply:
sure -- it's moot since the packet should never be decoded in the first place, but no harm in making it extra clear that this is not a valid packet